src/dlgTriggerEditor.cpp

```diff
@@ -4613,7 +4655,7 @@ void dlgTriggerEditor::slot_var_selected(QTreeWidgetItem *pItem)
     if (varType == LUA_TTABLE || varType == LUA_TFUNCTION)
     {
         // Edbee doesn't have a readonly option, so I'm using setEnabled
-        mpSourceEditorEdbee->setEnabled(false);
+        // mpSourceEditorEdbee->setEnabled(false);
```

Comment:
Some times the editor displays stuff that __should not be edited__ like _module names_ or is not something that __can have anything inserted__, like _lua functions_ or _tables_ in the "variable" view.